src/app/components/stacks-transaction-item/stacks-transaction-item.tsx

```diff
@@ -42,10 +57,13 @@ export function StacksTransactionItem({
   const { handleOpenStacksTxLink } = useStacksExplorerLink();
   const currentAccount = useCurrentStacksAccount();
 
-  const { pathname } = useLocation();
   const navigate = useNavigate();
   const { whenWallet } = useWalletType();
 
+  const cancelTransactionMatch = useMatch('/cancel-transaction/stx/:txid');
+  const increaseFeeMatch = useMatch('/increase-fee/stx/:txid');
```

Comment:
We have this in `IncreaseStxFee` in `route-urls`. Can you please use that and add `cancel-transaction` there also?